templates/registration/logged_out.html

```diff
@@ -0,0 +1,3 @@
+<h1>Logged out</h1>
+
+<a href="{% url 'login' %}">login again</a>

```

Comment:
A page with the only message about logging out feels empty. Add a base interface of the site on this page.